### DISCUSSION TITLE
Fritzbox zum verbinden mit kabelkopfstelle zwingen?

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#272] [Allgemein]
Hallo erst mal.

Also mir ist da was mit der fw 7.25, ich betreibe meine 6590 seid ca.1 Jahr als mesh client.
Bis zur Version 7.20 hat sich die box, mit der kabelkopfstelle syncronisiert und werte der Leitung angezeigt, mit der 7.25 wurde es schienbar geändert und im webinterface wird nur noch angezeigt, sie würde die Downstream Frequenz suchen.
Aber nur wen sie im mesh client modus ist, wen ich einstelle, das internet kommt per kabel, syncronisiert sie sich.

MEine frage ist, kann ich die box zwingen, sich zu verbinden?

**Comment by fda77** [comment]
Bei AVM bedeutet ein Update nicht unbedingt eine Verbesserung. Mal darf man keine backups mehr erstellen&einspielen, die Knöpfe am gerät sind nciht mehr nutzbar, Voip geht für manche Provider nicht mehr usw.
Daher könnte es sein dass es so gewollt ist. versuch da mal als master oder ganz ohne mess

**Reply by Conan179** [reply]
Als mesh master und ohne mesh, keine änderung, nur wen sie vom ip-client modus in den Kabelmodem modus umgestellt worden ist, syncronisiert sie sich, naja kann man nichts machen, war eh nur zu überwachung von störungen im Kabelnetz.

**Reply by fda77** [reply]
ip-client deaktiviert normalerweise das modem und routing komplett. Vielleicht war das vorher ein Fehler. Du könntest versuches es wieder über Bearbeiten der /var/flash/ar7.cfg zu aktivieren.
Was bringt es dir eigentlich im ip-client modus eine docsis-verbindung zu haben?

**Reply by Conan179** [reply]
> und werte der Leitung angezeigt,

> war eh nur zu überwachung von störungen im Kabelnetz.

Die Werte der einzelen Kannäle anzeigen und festhallten, ob eine störung im netz vorliegt.
![2021-04-16 23 12 22](https://user-images.githubusercontent.com/1906242/115084778-a1065180-9f09-11eb-9ed6-3ce1dfd17686.jpg)
Das ist von meiner 6591.

Ich werde mir mal die ar7.cfg anschauen.

**Reply by Conan179** [reply]
@fda77 danke für den Tipp mit der ar7.cfg, hab den punkt gefunden und aktiviert.
Doof ist, der multid gibt jetzt die ip von der 6590 als standartgateway und dns an und alle geräte die damit verbunden sind, haben dann kein internet mehr.
Aber das ist ne andere sache und hab es bereits aus gelagert #276

**Reply by fda77** [reply]
Welche Optionen hast du denn ändern müssen?
Einfach mit dem Befehl "route" eine eigene Default-Route anlegen. rc.custom sollte passen, ausser AVM überschreibt die später wieder.
Die Statistiken sind nicht schlecht. Wenn ich mal wieder Kabel hab will ich sowas auch!

**Reply by Conan179** [reply]
sync_groups {
                name = "sync_cable";
                enabled = no;
                media_type = media_type_cable;
                connect_delay_when_synced = 3s;
                pppoeiface_link_mode = link_mode_unknown;
                speed_in_netto = 1024;
                speed_out_netto = 512;
                manual_speed = no;
        } {
In dem Block die Zeile:
enabled = no;
auf
enabled = yes;

also 
cat /var/flash/ar7.cfg > /var/tmp/ar7.cfg
ar7.cfg per sftp geändert.
cat /var/tmp/ar7.cfg > /var/flash/ar7.cfg
dann noch
ar7cfgchanged
eingetippt und sie hat sich gesynct, dafür hat sich dann aber der avm_multid in den kopf gesetzt, den standartgatway und der dns zu sein und nicht mehr der mesh master.

wie mache ich das mit "route"?

**Reply by fda77** [reply]
Du kannst die Datei auch direkt auf der Box mit "nvi" bearbeiten. Am besten nochmal rebooten um sicher zu gehen das die Wert richtig abgespeichert wurden, manchmal schriebt AVm einen cache zurück.
 "route" wird wie auf jedem Gerät mit Linux benutzt

**Reply by Conan179** [reply]
Du ich komme mit vi/nvi nicht zurecht >.< ich nutzte normalerweise nano.

**Reply by fda77** [reply]
Das mach eigentlich nur die "cat" dazu: `cat $(which nvi)`

**Reply by Conan179** [reply]
Glauch ich dir, ich komme trozdem nicht mit vi zurecht 😅

**Reply by Conan179** [reply]
@fda77 ich hab jetzt "route add default gw 192.168.1.1 dsl" ausprobiert.
Bringt nichts, es wird zwar unter "route" angezeigt, aber geht trozdem nicht zum richtigen gateway weiter.

**Reply by fda77** [reply]
Nur anzeigen reicht nicht, es muss auch genutzt werden. Vielleicht gibts welche mit höherer Priorotät

**Reply by Conan179** [reply]
Wie mache ich das?

**Reply by fda77** [reply]
Mit "route"
